### ISSUE TITLE
PHPword: insert chart in template

### ISSUE THREAD

**Issue by clab8** [PHPOffice/PHPWord#1058]
I'm trying to modify a template with text and charts. How can I insert chart in my template as text value?
```
$templateProcessor = new \PhpOffice\PhpWord\TemplateProcessor('TemplateReport.docx');
$categories = array('A', 'B', 'C', 'D', 'E');
$series1 = array(1, 3, 2, 5, 4);
$chart = $section->addChart('doughnut', $categories, $series1);
$chart->getStyle()->setWidth(\PhpOffice\PhpWord\Shared\Converter::inchToEmu(5.5))->setHeight(\PhpOffice\PhpWord\Shared\Converter::inchToEmu(5));
$templateProcessor->setValue('chart', $chart);
$templateProcessor->saveAs('filename.docx);
```
'TemplateReport.docx' file contains ${chart} string

**Comment by troosan** [comment]
The ->setValue is just doing a text replace. I do not believe it is currently possible to insert an element in a template.

As it is written in the doc:
> It is not possible to directly add new OOXML elements to the template file being processed

**Comment by clab8** [comment]
I know, but I found how to insert image ([Add/Insert/Replace image in templates #550](https://github.com/PHPOffice/PHPWord/issues/550)) and how to insert a table ([Replace or add an table on a templates #271](https://github.com/PHPOffice/PHPWord/issues/271)) in template, nothing on charts.

Is there a way to do it?

**Comment by troosan** [comment]
Indeed, I guess this would have to be implemented. I haven't really had a look at the "reading" part of PHPWord but this feature looks very basic to me. Even the sample messes up the final document pretty badly. :-(

**Comment by it-delta** [comment]
Hi! This feature don`t work properly (
Please fix that )

**Comment by adminindolytics** [comment]
Hello,

I have template document word file and in that I need to add Chart so how I can add chart into it ?

File :
Some Text with ${setvalue} fields
Chart
Some Static text

Code :
$my_template = new \PhpOffice\PhpWord\TemplateProcessor(public_path('template.docx'));
$setvalue = 'Mandatory';
$my_template->setValue('setvalue', $setvalue);
$my_template->saveAs(storage_path('final.docx'));

Thanks in advance for your help.
